### DISCUSSION TITLE
No headphone output for LyraT 4.3 board

### DISCUSSION THREAD

**Discussion by youqi66** [pschatzmann/arduino-audiokit#99] [Q&A] [answered]
I tried the examples for AudioKit on LyraT 4.3 board, no headphone output. Debug info shows HEDPHONE_DETECT=-1 even if head phone is plugged in. The cause of it lies in lyrat_v4_3.h. 

Line 72: #define HEADPHONE_DETECT 19
Line 77: #define HEADPHONE_DETECT -1

Line 77 override the value of HEADPHONE_DETECT and set it to -1. 
After remote this line, the headphone output is correct.

**Comment by pschatzmann** [comment]
Can I expect a pull request ?

**Reply by youqi66** [reply]
issue is created

**Comment by youqi66** [comment]
pull request is generated
    On Tuesday, August 15, 2023 at 04:49:32 PM EDT, Phil Schatzmann ***@***.***> wrote:  
 
 


Can I expect a pull request ?

**Comment by pschatzmann** [comment, marked as answer]
Correction has been merged
